docs/features/searching.rst

```diff
@@ -32,9 +32,12 @@ For example, to enable search filtering by Blog or by Products in addition to Al
 
 Would enable the following filter options on the search page: All Results, Blog, Products.
 
+
 Search fields
 -------------
 
+.. deprecated:: 0.25, instead the `Wagtail search parameters <https://docs.wagtail.org/en/stable/reference/contrib/searchpromotions.html#module-wagtail.contrib.search_promotions>`_ should be used.
```

Comment:
I believe after the version number, the text should be on a separate line and indented. E.g.
```
.. deprecated:: 0.25

   This is the message.
```